arangod/ClusterEngine/ClusterIndex.cpp

```diff
@@ -502,7 +513,6 @@ ClusterIndex::coveredFields() const {
     case TRI_IDX_TYPE_GEO2_INDEX:
     case TRI_IDX_TYPE_FULLTEXT_INDEX:
     case TRI_IDX_TYPE_TTL_INDEX:
-    case TRI_IDX_TYPE_ZKD_INDEX:
```

Comment:
Not sure if the removal is correct here.
Otherwise, if we have a ZKD index that does not cover anything, it will now return `_fields` whereas before it returned `[]`.